ros/src/computing/perception/detection/packages/enet_image_segmenter/README.md

```diff
@@ -0,0 +1,28 @@
+# How to install Caffe for ENet
+
+1. Complete the [pre-requisites](http://caffe.berkeleyvision.org/install_apt.html) 
+
+2. Clone ENEt Caffe fork in your home directory
+```
+% git clone https://github.com/TimoSaemann/ENet.git
```

Comment:
Since caffe-enet is contained in submodule, recursive option is necessary.

`git clone --recursive https://github.com/TimoSaemann/ENet.git`

Reply:
@yk-fujii Done